.buildbot.sh

```diff
@@ -22,6 +22,7 @@ sh rustup.sh --default-host x86_64-unknown-linux-gnu \
 export PATH=`pwd`/.cargo/bin/:$PATH
 
 git clone --recurse-submodules --depth 1 https://github.com/softdevteam/yk
+export YKB_YKLLVM_BUILD_ARGS="define:CMAKE_C_COMPILER=/usr/bin/clang,define:CMAKE_CXX_COMPILER=/usr/bin/clang++"
```

Comment:
In general I have a preference for scoping these vars just to avoid unexpected consequences where (say) a Makefile does `cargo build` and picks this up without knowing it. So should we do:

```sh
cd yk
YKB_YKLLVM_BUILD_ARGS="..." cargo build
cd ..
```

?

Reply:
Try 90b018b